### PR TITLE
Kddimitrov/track command options

### PR DESCRIPTION
**Pull request by KristianDD**
## PR Checklist

- [ ] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [ ] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [ ] You have signed the [CLA](http://www.nativescript.org/cla).
- [ ] All existing tests are passing: https://github.com/NativeScript/nativescript-cli/blob/master/CONTRIBUTING.md#contribute-to-the-code-base
- [ ] Tests for the changes are included.

## What is the current behavior?

We don't have tracking for command options

## What is the new behavior?

We do have tracking for command options

Fixes/Implements/Closes #[Issue Number].

### CODE REVIEW

File: lib/declarations.d.ts [hunk 1]
```diff
@@ -910,6 +910,10 @@ interface IAndroidBundleValidatorHelper {
 	validateRuntimeVersion(projectData: IProjectData): void
 }
 
+interface IOptionsTrackHelper {
```
[the remark this answers is not in the document]
Comment (KristianDD) [reply lib/declarations.d.ts]: Good point. Will change it. :)

File: lib/common/declarations.d.ts [hunk 2]
```diff
@@ -1269,6 +1269,10 @@ interface IDashedOption {
 	 * Type of the option. It can be string, boolean, Array, etc.
 	 */
 	type: string;
+	/**
+	 * Should skip in tracking
+	 */
+	private?: boolean;
```
Comment (DimitarTachev) [inline lib/common/declarations.d.ts]: private -> hasSensitiveValue

**Comment by KristianDD** [comment]
run ci

File: lib/options.ts [hunk 3]
```diff
@@ -41,83 +41,85 @@ export class Options {
...
-			release: { type: OptionType.Boolean, alias: "r" },
-			var: { type: OptionType.Object },
-			default: { type: OptionType.Boolean },
-			count: { type: OptionType.Number },
-			analyticsLogFile: { type: OptionType.String },
-			hooks: { type: OptionType.Boolean, default: true },
-			link: { type: OptionType.Boolean, default: false },
-			aab: { type: OptionType.Boolean }
+			ipa: { type: OptionType.String, hasSensitiveValue: false },
+			frameworkPath: { type: OptionType.String, hasSensitiveValue: false },
```
Comment (DimitarTachev) [inline lib/options.ts]: hasSensitiveValue: true

File: lib/options.ts [hunk 4]
```diff
@@ -41,83 +41,85 @@ export class Options {
...
+			frameworkName: { type: OptionType.String, hasSensitiveValue: false },
+			framework: { type: OptionType.String, hasSensitiveValue: false },
+			frameworkVersion: { type: OptionType.String, hasSensitiveValue: false },
+			forDevice: { type: OptionType.Boolean, hasSensitiveValue: false },
+			provision: { type: OptionType.Object, hasSensitiveValue: true },
+			client: { type: OptionType.Boolean, default: true, hasSensitiveValue: false },
+			env: { type: OptionType.Object, hasSensitiveValue: false },
+			production: { type: OptionType.Boolean, hasSensitiveValue: false },
+			debugTransport: { type: OptionType.Boolean, hasSensitiveValue: false },
+			keyStorePath: { type: OptionType.String, hasSensitiveValue: false },
```
Comment (DimitarTachev) [inline lib/options.ts]: hasSensitiveValue: true

File: lib/options.ts [hunk 5]
```diff
@@ -41,83 +41,85 @@ export class Options {
...
+			keyStorePassword: { type: OptionType.String, hasSensitiveValue: true },
+			keyStoreAlias: { type: OptionType.String, hasSensitiveValue: true },
+			keyStoreAliasPassword: { type: OptionType.String, hasSensitiveValue: true },
+			ignoreScripts: { type: OptionType.Boolean, hasSensitiveValue: false },
+			disableNpmInstall: { type: OptionType.Boolean, hasSensitiveValue: false },
+			compileSdk: { type: OptionType.Number, hasSensitiveValue: false },
+			port: { type: OptionType.Number, hasSensitiveValue: false },
+			copyTo: { type: OptionType.String, hasSensitiveValue: false },
```
Comment (DimitarTachev) [inline lib/options.ts]: hasSensitiveValue: true

File: lib/options.ts [hunk 6]
```diff
@@ -41,83 +41,85 @@ export class Options {
...
+			platformTemplate: { type: OptionType.String, hasSensitiveValue: false },
```
Comment (DimitarTachev) [inline lib/options.ts]: hasSensitiveValue: true?

**Pull request merged**